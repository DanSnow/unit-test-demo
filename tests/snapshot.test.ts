import { test, expect } from 'vitest'

test('snapshot', () => {
  expect({
    a: 1,
    b: 2,
    c: 3,
  }).toMatchSnapshot()

  expect({
    a: 1,
    b: 2,
    c: 3,
  }).toMatchInlineSnapshot(`
    {
      "a": 1,
      "b": 2,
      "c": 3,
    }
  `)
})
