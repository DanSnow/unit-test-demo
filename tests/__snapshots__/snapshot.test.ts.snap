// Vitest Snapshot v1

exports[`snapshot 1`] = `
{
  "a": 1,
  "b": 2,
  "c": 3,
}
`;
